ext/mysqli/mysqli_driver.c

```diff
@@ -26,54 +26,35 @@
 #include "php_mysqli_structs.h"
 #include "zend_exceptions.h"
 
-#define MAP_PROPERTY_MYG_BOOL_READ(name, value) \
-static int name(mysqli_object *obj, zval *retval, bool quiet) \
-{ \
-	ZVAL_BOOL(retval, MyG(value)); \
-	return SUCCESS; \
-} \
 
-#define MAP_PROPERTY_MYG_BOOL_WRITE(name, value) \
-static int name(mysqli_object *obj, zval *value) \
-{ \
-	MyG(value) = Z_LVAL_P(value) > 0; \
-	return SUCCESS; \
-} \
-
-#define MAP_PROPERTY_MYG_LONG_READ(name, value) \
-static int name(mysqli_object *obj, zval *retval, bool quiet) \
-{ \
-	ZVAL_LONG(retval, MyG(value)); \
-	return SUCCESS; \
-} \
-
-#define MAP_PROPERTY_MYG_LONG_WRITE(name, value) \
-static int name(mysqli_object *obj, zval *value) \
-{ \
-	MyG(value) = Z_LVAL_P(value); \
-	return SUCCESS; \
-} \
+/* {{{ property driver_report_read */
+static int driver_reconnect_read(mysqli_object *obj, zval *retval, bool quiet)
```

Comment:
```suggestion
static zend_result driver_reconnect_read(mysqli_object *obj, zval *retval, bool quiet)
```
But it might make sense to change it to `void` as it always returns `SUCCESS`, or drop the out param and make it return the boolean value.

Reply:
I honestly don't know if I can make this change. Changing the return type seems like it could affect the caller. I made it the same type as all other property accessor methods. I know it makes no sense to unconditionally return SUCCESS here but in other properties we return also FAILURE. Bear in mind that all these functions are called generically from here https://github.com/php/php-src/blob/master/ext/mysqli/mysqli.c#L313

Reply:
Ah so this explains that, changing the return type should be fine as zend_result is an enum of SUCCESS and FAILURE to ensure it is used correctly, so if it's only called generically from that place, it should be fine.

Reply:
Is this what you had in mind?

Reply:
Yes :-)

Reply:
I'm sorry, but making the methods return void is a little too much for this PR. Maybe we can refactor it later. For the moment I changed them all to return zend_result.